### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by matiasgaratortiz**
Minimal Project Setup for Checkpotify:
- Config package.json
- Config bower.json and .bowerrc
- Config gulpfile.js
- Create folder structure

### CODE REVIEW

File: .bowerrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"directory": "vendor"
+}
```
Comment (leolower) [inline .bowerrc]: You should configure your editor to add new lines at the end of files.

Sublime:
```
	"ensure_newline_at_eof_on_save": true,
```

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	return gulp.src('src/app/js/**/*.js')
+		.pipe(jshint())
+		.pipe(jshint.reporter('jshint-stylish'));
+});
+
+/* compile less files */
+gulp.task('lr-css', function() {
+	return gulp.src('src/app/less/**/*.less')
+		.pipe(less())
+		.pipe(gulp.dest('src/css'))
```
Comment (leolower) [inline gulpfile.js]: Consider moving the built code outside of `src/`. Probably you can add a `build` directory at the same level or `src`.

Comment (leolower) [inline gulpfile.js]: Also remember to ignore it.

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+var gulp  = require('gulp'),
+    gutil = require('gulp-util'),
+    browserSync = require('browser-sync').create(),
+    image = require('gulp-image'),
+    jshint = require('gulp-jshint'),
+    less = require('gulp-less');
+
+
+/* Static server */
+gulp.task('serve', ['less'], function() {
```
Comment (unjapones) [inline gulpfile.js]: Doesn't this depend also on the tasks?:
* `js`
* `html`
* `image`

**Pull request merged**